src/components/InstancesList/InstancesList.js

```diff
@@ -123,6 +123,7 @@ class InstancesList extends React.Component {
     location: PropTypes.shape({
       search: PropTypes.string,
       state: PropTypes.object,
+      pathname: PropTypes.string,
```

Comment:
Could you check if it is possible to further increase the coverage on this file?